lisa/transformers/kernel_source_installer.py

```diff
@@ -336,6 +408,10 @@ def get_source_code(self) -> PurePath:
 
         return code_path
 
+    def get_kconfig_file(self) -> str:
```

Comment:
This function can be implemented in base and inherited

Reply:
Removed this function in latest push as this param is pushed at installer level schema